contracts/forwarder/SmartWallet.sol

```diff
@@ -291,13 +293,15 @@ contract SmartWallet is IForwarder {
                     success,
                     "initialize(bytes) call in logic contract failed"
                 );
+
+                bytes memory logicCell = abi.encodePacked(logic);
```

Comment:
Given that this is just one parameter, what would be the difference of doing this encoding?